my_scripts/cpu_batch

```diff
@@ -8,9 +8,12 @@ PROB_SIZE=$5
 NUM_ITERS=$6
```

Comment:
Instead of modifying this file, could you create a new file and just call it `event_based_cpu_batch` or something like that ?

Reply:
makes sense!